src/cursor/cur_file.c

```diff
@@ -694,10 +694,14 @@ __curfile_cache(WT_CURSOR *cursor)
     WT_SESSION_IMPL *session;
 
     cbt = (WT_CURSOR_BTREE *)cursor;
+    CURSOR_API_CALL(cursor, session, ret, cache, CUR2BT(cbt));
```

Comment:
As part of the `__wt_cursor_cache` function, the `cursor_cache` gets updated whenever it is failed. This change is not required.

Reply:
@kommiharibabu 
Adding this is to maintain consistency with other interfaces, which can improve some information, such as:
https://github.com/wiredtiger/wiredtiger/blob/develop/src/cursor/cur_file.c#L356

thanks
:)

Reply:
I understand your point. Then it is better to remove the stat increment inside the `cursor_cache` function. This function is called only from one place.

After going through the code, even if there is an error, that is possible with `cursor_reset` that is called inside the `cursor_cache` function, and the caching is performed. In that case, I feel there is no need for this stat itself?

Reply:
@kommiharibabu 
Thank you for your detailed explanation, I have change the code again.
thanks.
:)